code/datums/elements/art.dm

```diff
@@ -46,12 +46,11 @@
 	to_chat(user, span_notice("You start appraising [source]..."))
 	if(!do_after(user, 2 SECONDS, target = source))
 		return
-	var/mult = 1
+	var/mult
 	if(isobj(source))
```

Comment:
Instead of checking for obj, why not check for using integrity here instead? Integrity is on the atom, which is what ``source`` is, no need for this obj stuff or the base mult of 1 not just being on the var